src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -26,6 +26,7 @@
 @SuppressWarnings("WeakerAccess")
 public class Main extends GuiApplication {
 
+  // TODO Issue-662 - sync these values with release.yml build.gradle
```

Comment:
Here and in the other two spots: I recommend linking the full URL just in case this gets forked, e.g.:

// TODO: Sync these values with release.yml and build.gradle (https://github.com/BrightSpots/rcv/issues/625)